lib/nodes.py

```diff
@@ -56,6 +56,23 @@ def mark_online(node, now):
     node['flags']['online'] = True
 
 
+def mark_uplinks(nodeinfo, node):
+    mark_fastd_uplinks(nodeinfo, node)
+
+
+def mark_fastd_uplinks(nodeinfo, node):
```

Comment:
@tcatm suggested to use singular ( mark_fastd_uplink() ), as there is only one mark per node